src/server/server_family.cc

```diff
@@ -753,9 +753,9 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
   // Server metrics
   AppendMetricHeader("version", "", MetricType::GAUGE, &resp->body());
   AppendMetricValue("version", 1, {"version"}, {GetVersion()}, &resp->body());
-  AppendMetricHeader("role", "", MetricType::GAUGE, &resp->body());
-  AppendMetricValue("role", 1, {"role"}, {m.is_master ? "master" : "replica"}, &resp->body());
```

Comment:
Ok to remove - but please cc control plane team when removing/changing existing metrics.

Reply:
sure, I thought you guys switched to the news approach 4 months after :)